docs/project-setup/filesystem-sync.md

```diff
@@ -1,41 +1,22 @@
-of a build becomes challenging when your code base is on a volume mounted NFS share.
+Outrigger uses [Unison](https://github.com/bcpierce00/unison) to provide high-performance, bi-directional file synchronization between your local operating system and Docker containers.
 
-NFS also has the downside of not propagating filesystem modification events. This can be problematic if you want to run
-a process in your container to rebuild or update compiled project assets when modification notifications are triggered
-due to local file editing in your IDE.
+From the root of your project, all the files are synced via a dedicated container and exposed as a named [Docker Volume](https://docs.docker.com/engine/tutorials/dockervolumes/) to the rest of your application. This volume is used as the file "source" for a volume mount to allow nearly native filesystem performance and features.
 
-In order to better support these types of operations, Outrigger provides support for multi-directional filesystem syncing 
-(which includes filesystem notifications) using `unison`.  Your project root folder is synced via `unison` into a sync 
-container and exposed as a Docker Volume to the rest of your application. This volume is then used as your mount point 
-instead of the NFS share and allows nearly native filesystem performance and features.
+This is not used as the only means of sharing code or data into the filesystem, but it is the recommended approach for sharing primary application code that must also be readily available to developers and their local tools (e.g., IDEs).
 
-## Sync Volume Name
+For more on why Outrigger introduced Unison, jump down to [Why Sync Instead of NFS?](#why-sync-instead-of-nfs)
 
-The name of the sync volume can be set in a variety of ways. It is determined using the following precedence:
+## Setting Up Sync for Your Project's Containers
 
-1. Argument provided to the `project sync:start` command
-2. Specified in the `sync` -> `volume-name` property of your [Outrigger Project Configuration](./project-configuration.md)
-3. In your `docker-compose.yml` file, specified as an `external` volume with a name of the pattern `*-sync`
-4. If not specified anywhere else, it will use the name of the current project folder with `-sync` appended
-
-We recommend using approach #3, as it is the most explicit and is in line with all of the configuration you will need
-to take full advantage of `unison` syncing.
+To setup `unison` sync for your containers you will need to [reference an external volume](https://docs.docker.com/compose/compose-file/#volume-configuration-reference)
+in your `docker-compose.yml` and use that external volume in the mount specification for any services that need to
+reference the same set of files. Using the same volume will ensure all containers stay in sync. This includes specifying that volume for your [Build container](/common-tasks/using-the-build-container), if you are using that (via build.yml).
```

Comment:
Before the "Using the same volume " part it would be helpful to add some naming information. Suggestion:

Since volumes are global within your Docker host, the name of this external volume should be name-spaced to your project. The convention is to use projectname-sync as the volume name. Using the same volume will ensure all of your application's containers stay in sync. This includes ....